nikola/post.py

```diff
@@ -312,14 +343,28 @@ def _has_pretty_url(self, lang):
 
     @property
     def is_mathjax(self):
```

Comment:
Should we keep a `is_mathjax` dummy which logs that an update needs to be done to the theme, and returns the new `has_math` value? Otherwise, we might break some custom themes.

Reply:
This is v8, feel free to break themes. (They should switch to `math_helper.tmpl` anyways.)

Reply:
Done.